v1/nix/modules/drv-parts/nixpkgs-overrides/interface.nix

```diff
@@ -14,6 +14,12 @@ in {
       description = "Attributes we do not want to copy from nixpkgs";
     };
 
+    from = l.mkOption {
+      type = t.nullOr t.package;
+      description = "package from which to extract the attributes";
+      default = config.deps.python.pkgs.${config.name} or null;
```

Comment:
Is a python-specific default a good idea here?

This could be a good example of the need for `depsModules`or so in drv-parts?

Reply:
currently this module contains python specific logic in several places. We should split it into `nixpkgs-overrides` and `nixpkgs-overrides-python` at some point I guess. I'd say we better do this in another PR.

Reply:
Another PR is fine with me, but we should not forget it. I think the default isn't even used atm? But on second reading it's also no hard dependency on config.deps.python, so I don't care too much.